source/deployment/enterprise-deployment-guide.rst

```diff
@@ -0,0 +1,216 @@
+###########################
+Enterprise Deployment Guide
+###########################
+
+================
+About This Guide
+================
+
+********
+Audience
+********
+
+IT Leaders, System Administrators, and/or Project Managers
+
+.. Note::
+    Some features described in this guide are only available to Enterprise Edition E10 and/or E20. We will highlight specific licenses needed whenever applicable.
+
+*******************
+Learning Objectives
+*******************
+
+This guide provides foundational information necessary when developing a plan for a proof of concept and/or production-level enterprise deployment of the Mattermost on-premises edition. 
+
+It will not dive into the technical implementation, but will reference any relevant documentation.
+
+By the end of this guide you should have an understanding of the high-level requirements for a successful Mattermost Enterprise deployment. 
+
+This includes:
+
+- Choosing how to deploy the Mattermost Server application
+- Migrating from other ChatOps platforms
+- Integrating Mattermost with existing Single sign-on (SSO) providers
+- First steps/best practices for onboarding users
+
+*******
+Support
+*******
+
+If at any point you need additional help, we’re ready to assist, just reach out using any of these methods:
+
+- `Community <https://community.mattermost.com/>`_ - Our entire organization uses Mattermost, plus you’ll have the support of one of the best open source communities around
+- Documentation - We link to many parts of our `administrator  documentation <https://docs.mattermost.com/guides/administrator.html>`_ in this guide but encourage you to take a look at `all our documentation <https://docs.mattermost.com>`_
+- Enterprise Support  - If you’re an Enterprise Edition subscriber, you may open a support ticket in the Enterprise Edition Support portal
+
+.. Recommendation::
+    Visit the `Mattermost Community server <https://community.mattermost.com/>`_ and create an account. Join channel ..::TODO Specific Guide Support Channel?::.. to not only experience Mattermost right away but also for support if you get stuck at any point in this guide.
```

Comment:
If the channel is not yet available, propose removing this section or creating a relevant channel.